programs/uxd/src/instructions/edit_controller.rs

```diff
@@ -139,12 +186,13 @@ impl<'info> EditController<'info> {
 
         // Validate the depositories_routing_weight_bps if specified
         if let Some(depositories_routing_weight_bps) = fields.depositories_routing_weight_bps {
-            let total_weight_bps = depositories_routing_weight_bps
-                .identity_depository_weight_bps
-                .checked_add(depositories_routing_weight_bps.mercurial_vault_depository_weight_bps)
-                .ok_or(UxdError::MathOverflow)?
-                .checked_add(depositories_routing_weight_bps.credix_lp_depository_weight_bps)
-                .ok_or(UxdError::MathOverflow)?;
+            let total_weight_bps = checked_add(
```

Comment:
any checked_sum in existance? prob not but 😆

Reply:
Didnt find any :p